style.css

```diff
@@ -82,3 +87,331 @@
   padding-left: 0;
   padding-top: 20px;
 }
+
+.works {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 114px 24px;
+  width: 375px;
+  height: 2788px;
+  background: #f7f7f9;
+}
+
+.cardworks1 {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  border-radius: 16px;
+  background: #fff;
+  padding: 16px;
+  height: 670px;
+  width: 350px;
+}
```

Comment:
- [ ] Please check all your project cards and ensure that the designs are same with the Figma design I have provided below:

|Your Design| Figma Design|
|:--------------:|:-----------------:|
|![image](https://user-images.githubusercontent.com/99715640/219574074-4d4eb25b-8c23-4155-95a5-61168972dcd1.png)|![image](https://user-images.githubusercontent.com/99715640/219574171-8d11d849-9a0a-49a7-a52f-efb27ba2292f.png)|

- To summarize, there seem to be no styles declared for the font properties, your list of skills is all underlined as against the required design and your buttons seem to need some alignment effects or equal padding to make sure the spaces are uniform :+1: